centml/values.yaml

```diff
@@ -1,20 +1,31 @@
 nodeSelector:
   region: LGA1
 
+affinity:
+  nodeAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution:
+      nodeSelectorTerms:
+        - matchExpressions:
+            - key: topology.kubernetes.io/region
+              operator: In
+              values:
+                - "{{.Values.nodeSelector.region}}"
```

Comment:
Shouldn't template in the values file. This should be in the template.

Reply:
Good catch! I fixed it in the latest commit!